src/main/java/org/opentripplanner/model/impl/OtpTransitServiceImpl.java

```diff
@@ -284,6 +288,10 @@ public ServiceCalendar getCalendarForServiceId(FeedScopedId serviceId) {
         throw new MultipleCalendarsForServiceIdException(serviceId);
     }
 
+    @Override
+    public Collection<Area> getAllAreas() {
```

Comment:
Can we rename these methods and the class to use FlexArea rather than just Area?